docs/modules/user-guide/partials/con_devfile.adoc

```diff
@@ -1,17 +1,17 @@
 [id="con_devfile_{context}"]
-= Overview of devfiles
+= About devfiles
 
 [role="_abstract"]
 
-.How devfiles work
+.How to use devfiles
 
-As a YAML file, devfiles simplify the development process. Devfiles tell you the steps you must take to get your application up and running from a source code to an actual product.When you download a devfile, it assesses your application development lifecycle, then provides you strategies and best practices for end-to-end application development. Devfiles do not force you to use new workflows nor tools, but rather, provide you instructions to automate your current, manual processes. For example, a devfile can take the repository hosting your application source code, build the code, run the application on your local container, then deploy the application to cloud-native containers. Unlike many development tools, devfiles can build, run, and deploy, automating many of your manual processes and simplifying your end-to-end development.
+Devfiles are YAML files, and they simplify your development process. In your IDE, devfiles tell you which best practices to apply to take your development project from source code to running application. Using the instructions given by devfiles, you can automate your current, manual processes. A devfile can take in the repository hosting your application source code. You can use the devfile to build your code, run your application on your local container, and then deploy your application to cloud-native containers.
 
 .Features and benefits of devfiles
 
-Devfiles provide guidelines and instructions for enhanced development processes. Devfiles uses the command line interface (CLI)  tool, odo, to execute the actions: build run, deploy. By using odo, devfiles provide consistent patterns of behavior for applications written in different languages and create workspaces composed of multiple containers. In these containers, you can make any number of identical workspaces from the same devfile. Creating multiple workspaces lets you share your devfile with different teams. Sharing a single devfile across multiple teams, ensures that each team has the same user experience and the same build, run, deploy behaviors.
+Devfiles give guidelines and instructions for enhanced development processes. Devfiles use the command line interface (CLI)  tool, `odo`, to build run, and deploy your application. By using odo, devfiles provide consistent patterns of behavior for applications written in different languages and create workspaces composed of multiple containers. In these containers, you can create any number of identical workspaces from the same devfile. If you create multiple workspaces, you can share your devfile with different teams. By sharing a single devfile across multiple teams, you can ensure that each team has the same user experience and the same build, run, deploy behaviors.
```

Comment:
SMEs:

Another good question from doc review, 
```does a devfile tell me best practices or does it automate software builds? Or both?```

I think it does both...but I'm also not sure how a devfile can on one hand give users suggestion for end-to-end development, but on the other hand, devfiles can build, run, and deploy and automate manual processes.

Reply:
This section sounds like only `odo` uses devfiles. `odo` is just one of the examples of the tools that use it, therefore, our wording should reflect that.

For the SME question above, devfile captures the best practices and the steps for doing the development. Devfiles itself is not a tool so it does not execute those build and run, etc. It is the tools that translates the best practices and automation steps and execute them during development.

Reply:
Thanks, Elson, made the change.

Any SME reviewer, please see if I addressed this concern in the latest commit, thanks!